virttest/block.py

```diff
@@ -0,0 +1,21 @@
+"""
+Utility functions to handle block related actions.
+
+"""
+
+from . import qemu_monitor
+
+
+def get_block_virtual_size(dev_name):
+    """
+    Get block virtual-size of specified block device with dev_name.
+
+    :param dev_name: block device name, like "drive-hotadd" etc.
+    :return: Matched block device virual size.
+    """
+    blocks_info = qemu_monitor.info("block")
+    for dict_block in blocks_info:
+        if dev_name == dict_block['device']:
+            size = dict_block['inserted']['image']['virtual-size']
+            return int(size)
+    return None
```

Comment:
it will return None by default, so I think no need to add \"return None\" in the end

Reply:
oo, thanks, will update it~